routes/api/users.js

```diff
@@ -93,4 +93,8 @@ router.get('/:username/social/', async (req, res) => {
   });
 });
 
+router.post('/:id/follow', verifyJwt(), FollowController.follow);
```

Comment:
@NkFab Try to use meaning full request parameters such as userId instead of just id